ofl/notokufiarabic/METADATA.pb

```diff
@@ -1,5 +1,5 @@
 name: "Noto Kufi Arabic"
-designer: "UNKNOWN"
+designer: "Google LLC"
```

Comment:
While "Google LLC" is the legal name, "Google" is the brand name, and should be used here (and therefore what is shown in catalog UI)